packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -430,13 +430,20 @@ export default defineComponent({
       this.setSidebarPanel('versions-item')
     },
     async loadData() {
+      if (this.file.type === 'folder' || this.isPublicLinkContext) {
+        return
+      }
       this.loading = true
-      const calls = []
-      if (this.file.type === 'file' && isUserContext(this.$router, this.$route)) {
-        calls.push(this.loadVersions({ client: this.$client, fileId: this.file.id }))
+      try {
+        await Promise.allSettled([
```

Comment:
Why allSettled? Can the catch block be entered at all then?

Reply:
hm true, probably not...

Reply:
Running into a major scope creep... reverting that change and following up in a separate PR.

Reply:
👍🏻